native/build.gradle

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+plugins {
+    id 'java'
+    id 'checkstyle'
+    id 'com.github.spotbugs'
+    id 'jacoco'
+}
+
+description = 'Ballerina - Observability Natives'
+
+repositories {
+    mavenLocal()
+    maven {
+        url = 'https://maven.wso2.org/nexus/content/repositories/releases/'
+    }
+
+    maven {
+        url = 'https://maven.wso2.org/nexus/content/groups/wso2-public/'
+    }
+
+    maven {
+        url = 'https://repo.maven.apache.org/maven2'
+    }
+
+    maven {
+        url = 'https://maven.pkg.github.com/ballerina-platform/*'
+        credentials {
+            username System.getenv("packageUser")
+            password System.getenv("packagePAT")
+        }
+    }
+}
```

Comment:
We don't need to add this to all the submodules again since we have added this once within the `allprojects` section in the project root.

Let's also remove any other duplicated parts already mentioned in the `allprojects` section.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-observe/pull/1/commits/ed8df9f5f8ca5121b0dd0a3a15cb43f5ed96749a